requirements.txt

```diff
@@ -7,7 +7,7 @@ attrs==19.1.0
 jsonpickle==1.1
 redis==3.2.1
 fakeredis==1.0.3
-pymongo[tls,srv]==3.8.0
+pymongo==3.8.0
```

Comment:
Why do we need that?

Reply:
not a change that i did, it was from previous commit